### PR TITLE
Change SQL built-in function's symbol

### PR DESCRIPTION
**Pull request by chenhao7253886**

### CODE REVIEW

File: gensrc/script/gen_vector_functions.py [hunk 1]
```diff
@@ -410,7 +410,7 @@
 #ifndef BDG_PALO_OPCODE_VECTOR_FUNCTIONS_H\n\
```
Comment (wuyunfeng) [inline gensrc/script/gen_vector_functions.py]: BDG_PALO_OPCODE_VECTOR_FUNCTIONS_H -> DORIS_OPCODE_VECTOR_FUNCTIONS_H

File: gensrc/script/gen_functions.py [hunk 2]
```diff
@@ -595,7 +595,7 @@
 #ifndef BDG_PALO_OPCODE_FUNCTIONS_H\n\
```
Comment (wuyunfeng) [inline gensrc/script/gen_functions.py]: BDG_PALO_OPCODE_FUNCTIONS ->DORIS_OPCODE_FUNCTIONS

File: gensrc/script/gen_build_version.sh [hunk 3]
```diff
@@ -143,7 +143,7 @@ cat >"${GEN_CPP_DIR}/version.h" <<EOF
 #ifndef PALO_GEN_CPP_VERSION_H
 #define PALO_GEN_CPP_VERSION_H
 
-namespace palo {
+namespace doris {
 
 #define PALO_BUILD_VERSION "${build_version}"
```
Comment (wuyunfeng) [inline gensrc/script/gen_build_version.sh]: PALO_BUILD_VERSION -> DORIS_BUILD_VERSION

**Review by morningman — CHANGES_REQUESTED** [review]
You should describe what is changed in your commit log body.

**Pull request closed without merging**